.gitignore

```diff
@@ -1,4 +1,10 @@
 openapitools.json
 node_modules/
 package-lock.json
-package.json
\ No newline at end of file
+package.json
+
+#Python
+python/
+# Environments
+*env*
+*ENV*
```

Comment:
New line at the end of the file.

I believe ENV is not needed here and venv is also recommended 😉 .

Reply:
`*env*` already contains `venv` right? 🤔

Reply:
yeah, my bad.